api/Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM amsterdam/python:3.8-buster
 MAINTAINER datapunt@amsterdam.nl
 
 ENV PYTHONUNBUFFERED 1
```

Comment:
minor: `PYTHONUNBUFFERED 1` staat al in de `amsterdam/python:3.8-buster` image dus is opzich niet nodig hier